packages/dd-trace/test/plugins/util/ci.spec.js

```diff
@@ -0,0 +1,44 @@
+const { getCIMetadata } = require('../../../src/plugins/util/ci')
+const jenkinsEnv = require('./ci-env/jenkins.json')
+const gitlabEnv = require('./ci-env/gitlab.json')
+const circleciEnv = require('./ci-env/circleci.json')
+const githubEnv = require('./ci-env/github.json')
```

Comment:
can you load all json files in `ci-env` directory?

Reply:
yeah I'll do that in a different PR when I add support for less known CIs. I didn't want to make this PR even bigger :)